lib/snapshot.go

```diff
@@ -2000,21 +2008,21 @@ func (status *SnapshotStatus) Initialize(snapshotDb *badger.DB, snapshotDbMutex
 
 func (status *SnapshotStatus) ToBytes() []byte {
 	var data []byte
-	data = append(data, UintToBuf(status.MainDBSemaphore)...)
-	data = append(data, UintToBuf(status.AncestralDBSemaphore)...)
+	data = append(data, UintToBuf(0)...) // Formerly status.MainDBSemaphore
```

Comment:
I left this here for backward compatibility, but I guess we could simplify if we really want.